op-acceptance-tests/tests/interop/proofs/challenger_test.go

```diff
@@ -0,0 +1,39 @@
+package proofs
+
+import (
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl/proofs"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+)
+
+func TestChallengerPlaysGame(gt *testing.T) {
+	// Setup
+	t := devtest.SerialT(gt)
```

Comment:
Can this be `detest.ParallelT(gt)` so the test can run in parallel with others?

Reply:
yep - good spot